index.rst

```diff
@@ -221,5 +246,128 @@ Banging on the side of the strut to excite some kind of bending mode was less su
 
 .. image:: ./_static/Accel_PSD_Side_Bang_20221004T132454Z.png
 
+Anemometer move - 22-Nov-22
+=================================
+
+Having the anemometer mounted at a fixed point in the dome proved to be not very useful.  So we moved the anemometer so that it was mounted on the telescope fork (see photo below). This way it is much closer to seeing the wind environment that the telescope sees.  At about the same time, the CSC was changed so that was is strored in the EFD, instead of being a stream of wind speeds, is the mean and standard deviation of the measurements every minute.  The above set of tests was repeated twice in the new configuration.
+
+.. image:: ./_static/On_telescope_22Nov22.jpg
+
+
+	   
+Fourth test - 23-Nov-22
+=================================
+
+The tests were repeated on 23-Nov-22.  The wind velocity outside the dome as measured by the Gemini weather station was 3.6 m/s from 302 degrees.  The slit was only about 50% open to prevent the sun from falling on the telescope.  The telescope and dome were aligned to 180 degrees.  The figure below shows the setup: 
+
+		
+.. image:: ./_static/Dome_Setup_23Nov22.jpg
+
+**Setup on 23-Nov-22**
+
+
+These images show the anemometer readings, the accelerometer readings, and the mount encoder data as the fan speed is ramped up:
```

Comment:
Do you have an information about the axes of the anemometer and how they roughly relate to the el/az?
Naively I would expect some sort of flow/turbulence in the elevation direction, which would correspond to one of the anemometer axes?